examples/jsm/exporters/DRACOExporter.js

```diff
@@ -18,6 +18,24 @@ import { Color } from 'three';
 /* global DracoEncoderModule */
 
 class DRACOExporter {
+	// Encoder methods
+
+	static MESH_EDGEBREAKER_ENCODING = 1;
+	static MESH_SEQUENTIAL_ENCODING = 0;
```

Comment:
I think we have decided some time ago _not_ to use static fields yet? /ping @Mugen87

Reply:
#25105

Reply:
We can't use them in pmndrs either, but the equivalent would be:

```js
@__PURE__ Object.assign( DRACOExporter, {

    MESH_EDGEBREAKER_ENCODING: 1;
    static MESH_SEQUENTIAL_ENCODING: 0;
} )
```

Reply:
b903a8fcec389a6303bae3198937b96dc1b7974e removes the usage. Can split this into a separate PR.